merge_state_block/release.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't like the idea of having to have a different directory for each block. Especially if that directory is never really used in the `Reference` or anything like that.

Can we support including multiple blocks and/or multiple specs in `release.json` and `spec.json` respectively? In other words, have a list of objects in those files, not just a single object.